packages/helix-shared-config/test/specs/queryconfigs/broken.json

```diff
@@ -2,7 +2,6 @@
   "indices": {
     "blog-posts": {
       "fetch": "https://${repo}-${owner}.project-helix.page/${path}",
-      "name": "blog-posts",
```

Comment:
The JSON representation now no longer contains the `name` entry, the array items in `indices` do, though.